modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -214,40 +214,51 @@ case, *Source IP* refers to the client making the request.
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Value |Description
-| *Max connections*
-| The maximum amount of simultaneous connections to allow.
+| *Max Connections*
+| The maximum number of simultaneous connections to allow.
 
 *Default value*: 512 connections
 
-| *Max requests per connection*
+| *Max Requests per Connection*
 | The maximum number of requests per connections to allow. +
-This value will determine how much reuse a connection can allow; consider the amount of CPU required to terminate and re-establish a TLS encrypted connection when lowering this value.
+Because this value determines how much reuse a connection allows, consider the amount of CPU required to terminate and reestablish a TLS-encrypted connection when lowering this value.
```

Comment:
when lowering > before lowering?
```suggestion
Because this value determines how much reuse a connection allows, consider the amount of CPU required to terminate and reestablish a TLS-encrypted connection before lowering this value.
```